main.go

```diff
@@ -102,19 +127,34 @@ func handleError(err error, w http.ResponseWriter) {
 func zipper(writer io.Writer, incoming chan RemoteFile, complete chan error) {
 	archive := zip.NewWriter(writer)
 	for rf := range incoming {
+
 		zipEntryHeader := &zip.FileHeader{
 			Name:   rf.path,
 			Method: zip.Deflate,
 		}
+
 		zipEntryHeader.SetModTime(time.Now())
```

Comment:
SetModTime is deprecated it seems. can use as below
```
zipEntryHeader.SetModTime(time.Now())
zipEntryHeader.Modified = time.Now().UTC()
```

Reply:
Nice catch. Changing it.